src/NUnitFramework/framework/Assert.Conditions.cs

```diff
@@ -201,33 +202,35 @@ public static void IsFalse(bool? condition)
         /// <param name="condition">The evaluated condition</param>
         public static void IsFalse(bool condition)
         {
-            Assert.That(condition, Is.False ,null, null);
+            Assert.That(condition, Is.False, null, null);
         }
 
         #endregion
 
         #region NotNull
 
+#pragma warning disable CS8777 // Parameter must have a non-null value when exiting.
+
         /// <summary>
         /// Verifies that the object that is passed in is not equal to <see langword="null"/>. Returns without throwing an
         /// exception when inside a multiple assert block.
```

Comment:
Note this documentation I added to explain why we decided not to add `[NotNull]`.